Admin.js

```diff
@@ -114,561 +136,771 @@ var Admin = {
+            "emptyclass": "editable-empty btn btn-sm btn-default",
+            "emptytext": "<i class=\"fa fa-pencil\"></i>",
+            "container": "body",
+            "placement": "auto",
+            "success" (response) {
+
+                const html = jQuery(response);
                 Admin.setup_xeditable(html);
-                jQuery(this)
-                    .closest('td')
-                    .replaceWith(html);
+                jQuery(this).
+                    closest("td").
+                    replaceWith(html);
+
             },
-            error: function(xhr, statusText, errorThrown) {
+            "error" (xhr, statusText, errorThrown) {
+
                 return xhr.responseText;
+
             }
         });
+
     },
 
     /**
-     * render log message
+     * Render log message
      * @param mixed
      */
-    log: function() {
-        var msg = '[Sonata.Admin] ' + Array.prototype.join.call(arguments,', ');
+    "log" () {
+
+        const msg = `[Sonata.Admin] ${Array.prototype.join.call(arguments, ", ")}`;
         if (window.console && window.console.log) {
+
             window.console.log(msg);
+
         } else if (window.opera && window.opera.postError) {
+
             window.opera.postError(msg);
+
         }
+
     },
 
     /**
      * NEXT_MAJOR: remove this function.
      *
      * @deprecated in version 3.0
      */
-    add_pretty_errors: function() {
-        console.warn('Admin.add_pretty_errors() was deprecated in version 3.0');
+    "add_pretty_errors" () {
+
+        console.warn("Admin.add_pretty_errors() was deprecated in version 3.0");
+
     },
 
-    stopEvent: function(event) {
+    "stopEvent" (event) {
+
         // https://github.com/sonata-project/SonataAdminBundle/issues/151
-        //if it is a standard browser use preventDefault otherwise it is IE then return false
-        if(event.preventDefault) {
+        // If it is a standard browser use preventDefault otherwise it is IE then return false
+        if (event.preventDefault) {
+
             event.preventDefault();
+
         } else {
+
             event.returnValue = false;
+
         }
 
-        //if it is a standard browser get target otherwise it is IE then adapt syntax and get target
-        if (typeof event.target != 'undefined') {
+        // If it is a standard browser get target otherwise it is IE then adapt syntax and get target
+        if (typeof event.target !== "undefined") {
+
             targetElement = event.target;
+
         } else {
+
             targetElement = event.srcElement;
+
         }
 
         return targetElement;
+
     },
 
-    add_filters: function(subject) {
-        Admin.log('[core|add_filters] configure filters on', subject);
-        jQuery('a.sonata-toggle-filter', subject).on('click', function(e) {
+    "add_filters" (subject) {
+
+        Admin.log("[core|add_filters] configure filters on", subject);
+        jQuery("a.sonata-toggle-filter", subject).on("click", (e) => {
```

Comment:
- eslint: Arrow function has too many statements (17). Maximum allowed is 10.
- eslint: Identifier name 'e' is too short (< 2).